docs/move-out-of-stock-products-to-the-end-of-a-collection/script.liquid

```diff
@@ -95,9 +95,46 @@
     {% endif %}
   {% endif %}
 
+  {% assign all_product_ids_current_sort = array %}
+  {% assign cursor = nil %}
+
+  {% for n in (0..100) %}
+    {% capture query %}
+      query {
+        collection(id: {{ collection.admin_graphql_api_id | json }}) {
+          products(
+            first: 250
```

Comment:
I believe that using sortKey: COLLECTION_DEFAULT is not needed here... same result without it, yes?

Reply:
<img width="315" alt="Screen Shot 2021-07-20 at 1 09 49 PM" src="https://user-images.githubusercontent.com/302012/126381749-29fb4599-afce-4fec-867e-eb9930913b1c.png">

It _is_ the default, but honestly I'd include it explicitly anyway, to establish direct contrast with the other collection->products query later in the task code.

Reply:
Updated